core/examples/luigi-example-svelte/src/routes/Home.svelte

```diff
@@ -10,7 +10,7 @@
   import LuigiClient from '@kyma-project/luigi-client';
   import { onMount } from 'svelte';
 
-  export let message;
+  export let message='Hello world'
```

Comment:
**Note**: _This message would be only visible if micro frontend runs without Luigi._